assets/src/scss/pages/search/_search-results.scss

```diff
@@ -61,6 +57,10 @@
           width: max-content;
         }
       }
+
+      .search-result-item-image:hover + * .search-result-item-headline {
```

Comment:
One tiny drawback is that `+` selectors depend on the elements being next to each other, which could be easy to overlook and break when changes to the markup are made. I guess using `~` is a little more robust. But then again changing the markup should involve checking dependent CSS, so feel free to ignore :shrug:

Reply:
Makes sense, specially for a feature easy to overlook :ok_hand: